lib/plugins/aws/deploy/compile/events/apiGateway/README.md

```diff
@@ -157,3 +157,40 @@ That wouldn't be required if you hadn't set the `private` property to `true`.
 Setting an API Gateway proxy can easily be done by adding two custom CloudFormation resource templates to your
 `serverless.yml` file. Check [this guide](https://github.com/serverless/serverless/blob/v1.0/docs/guide/custom-provider-resources.md)
 for more info on how to set up a proxy using custom CloudFormation resources in `serverless.yml`.
+
+### Enabling CORS for your endpoints
+To set CORS configurations for your HTTP endpoints, simply modify your event configurations as follows:
+
+```yml
+functions:
+  hello:
+    handler: handler.hello
+    events:
+      - http:
+        path: user/create
```

Comment:
This needs another 2 spaces for indentation.
YAML is nasty sometimes (I faced the exact same problem recently) and IMHO I would like this syntax more than adding 2 more spaces.

Here's an example of the difference between the two different indentations.

```
- http:
  path: users/create
  method: get
```

translates to:

```
[
  {
    "http": null,
    "path": "users/create",
    "method": "get"
  }
]
```

Whereas

```
- http:
    path: users/create
    method: get
```

translates to

```
[
  {
    "http": {
      "path": "users/create",
      "method": "get"
    }
  }
]
```